tests/Client.UiTests/Tests.fs

```diff
@@ -0,0 +1,43 @@
+module Client.UiTests.Tests
+
+open canopy.classic
+open Expecto
+open System
+open System.IO
+
+let serverUrl = Environment.GetEnvironmentVariable("SERVER_URL")
+
+type TypeInThisAssembly() =
+    class end
+
+let screenshotFolder = FileInfo(System.Reflection.Assembly.GetAssembly(typeof<TypeInThisAssembly>).Location).Directory.FullName
+
+let testCase name f =
+    testCase name (fun x ->
+        try
+            f x
+        with
+        | exn ->
+            screenshot screenshotFolder (name  + "-" + System.DateTime.Now.ToString("MMM-d_HH-mm-ss")) |> ignore
+            raise exn
+    )
+
```

Comment:
Because we are just starting, I suggest to remove all this stuff for now - we don't have reporting set up properly so those screenshots look strange...

Reply:
okay, it's just same as it was in safe app sample